0B.preprocess_plier_pathways.ipynb

```diff
@@ -616,6 +636,188 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(array([  1,   2,   3,   4,   5,   6,   7,   8,   9,  10,  11,  12,  13,\n",
+      "        14,  15,  16,  17,  18,  19,  20,  21,  22,  23,  24,  25,  26,\n",
+      "        27,  28,  29,  30,  31,  32,  33,  35,  36,  37,  38,  39,  40,\n",
+      "        41,  42,  43,  44,  45,  47,  48,  49,  50,  51,  52,  54,  55,\n",
+      "        56,  57,  60,  61,  62,  63,  64,  65,  66,  68,  71,  72,  73,\n",
+      "        76,  77,  80,  82,  83,  86,  88, 102, 103, 107, 110, 111, 122,\n",
+      "       136, 138, 139]), array([1346,  947,  708,  596,  426,  305,  279,  196,  176,  118,   90,\n",
+      "         86,   78,   89,   64,   52,   45,   35,   33,   37,   14,   23,\n",
+      "         13,   18,   30,    9,   11,    7,    6,    5,    5,   10,   13,\n",
+      "          3,    5,    9,    7,    5,    6,    3,    2,    4,    1,    4,\n",
+      "          7,   34,    6,    4,    3,    2,    4,    1,    3,    2,    2,\n",
+      "          2,    3,    1,    1,    1,    1,    2,    1,    2,    1,    2,\n",
+      "          2,    1,    1,    3,    1,    1,    1,    1,    1,    1,    1,\n",
+      "          1,    1,    2,    1]))\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
It isn't immediately clear what these two bar plots are showing. Are they the distribution of the number of genes in the pathways for the canonical data vs. the simulated data?